hack/clean/clean.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"flag"
+	"os"
+	"strings"
+	"time"
+
+	mgmtfeatures "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-07-01/features"
+	"github.com/sirupsen/logrus"
+
+	utillog "github.com/Azure/ARO-RP/pkg/util/log"
+	"github.com/Azure/ARO-RP/pkg/util/purge"
+)
+
+var (
+	dryRun = flag.Bool("dryRun", true, `Dry run`)
+)
+
+const (
+	defaultTTL          = 48 * time.Hour
+	defaultCreatedAtTag = "createdAt"
+	defaultKeepTag      = "persist"
+)
+
+func main() {
+	ctx := context.Background()
+	log := utillog.GetLogger()
+
+	flag.Parse()
+
+	if err := run(ctx, log); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func run(ctx context.Context, log *logrus.Entry) error {
+	subscriptionID := os.Getenv("AZURE_SUBSCRIPTION_ID")
+
+	var ttl time.Duration
+	if os.Getenv("AZURE_PURGE_TTL") != "" {
+		var err error
+		ttl, err = time.ParseDuration(os.Getenv("AZURE_PURGE_TTL"))
+		if err != nil {
+			return err
+		}
+	} else {
+		ttl = defaultTTL
+	}
+
+	var createdTag = defaultCreatedAtTag
+	if os.Getenv("AZURE_PURGE_CREATED_TAG") != "" {
+		createdTag = os.Getenv("AZURE_PURGE_CREATED_TAG")
+	}
+
+	purgeEnvironment := "" 
+	if os.Getenv("AZURE_PURGE_ENVIRONMENT") != "" {
+		purgeEnvironment = os.Getenv("AZURE_PURGE_ENVIRONMENT") 
+	}
```

Comment:
```suggestion
	purgeEnvironment := os.Getenv("AZURE_PURGE_ENVIRONMENT") 
```

No need to check if `AZURE_PURGE_ENVIRONMENT` is an empty string if that's supposed to be the default value of `purgeEnvironment` anyway.